src/index.html

```diff
@@ -9,10 +9,136 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.css"
     />
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div
+        class="calendar__day"
+        data-day="1"
+      ></div>
```

Comment:
Not fixed, remove all data attributes 
<img width="839" alt="image" src="https://github.com/user-attachments/assets/db773cc7-5e8e-44fd-9b08-a5c0d0d4d536">
```suggestion
      <div class="calendar__day"></div>
```